packages/phrase/src/phrase-api.ts

```diff
@@ -161,7 +160,7 @@ export async function pushTranslations(
       | undefined
     >(`uploads`, {
       method: 'POST',
-      body: formData,
+      body: formData.toString(),
```

Comment:
`node-fetch`'s `body` type accepts the type of `formData`, but native fetch's doesn't, so converting to string.